fastlane/Fastfile

```diff
@@ -44,26 +57,111 @@ end
 default_platform :ios
 
 platform :ios do
+  # Builds the app binary for Enterprise distribution (Prototype Build)
+  #
+  # @param fetch_code_signing [Boolean] If true, will call `configure_code_signing_enterprise` before building. Recommended on CI.
+  # @param build_number [String] The build number to use for the build. When in CI, defaults to the value of `BUILDKITE_BUILD_NUMBER` environment variable.
+  # @param version [String] The version to use for the build. When in CI, defaults to the PR number, falling back to the branch name.
+  #
+  lane :build_prototype do |fetch_code_signing: true, build_number: build_number_from_ci, version: pr_number_or_branch_name_from_ci|
+    if fetch_code_signing
+      configure_code_signing_enterprise
+    else
+      UI.message('Skipping fetching certificates and provisioning profiles as requested.')
+    end
+
+    build_app(
+      scheme: 'GravatarApp',
+      configuration: 'Enterprise',
+      # Explicitly set iOS destination to avoid xcodebuild accidentallypicking up tvOS
+      destination: 'generic/platform=iOS',
```

Comment:
In #7, I removed macOS as a supported architecture.

I considered removing tvOS as well because I don't think there are plans at the moment to support it. But tvOS comes from free, and this so far is the only extra line of code it requires to coexist in the project. So, I decided to leave it.

However, happy to remove it in the name of YANGI. Let me know.